tests/fixtures.py

```diff
@@ -82,7 +82,7 @@ def __init__(self):
         self.cache = {}
         self.elements = PreProcessedElements()
 
-    def generate(self, kind, n, t, arg1=None, arg2=None, k=1000):
+    def generate(self, kind, n, t, arg1=None, arg2=None, k=2000):
```

Comment:
I think it would be better to override this in your test cases-- I found a pattern like [this](https://github.com/initc3/HoneyBadgerMPC/pull/233/files#diff-5994374a4e4f9253d67655fb09c077ecR20) to work pretty well.